packages/fxa-settings/config/webpack.config.js

```diff
@@ -330,7 +330,10 @@ module.exports = function (webpackEnv) {
           'react-dom$': 'react-dom/profiling',
           'scheduler/tracing': 'scheduler/tracing-profiling',
         }),
-        fxaCryptoDeriver: require.resolve('@fxa/vendored/crypto-relier'),
+        fxaCryptoDeriver: path.resolve(
```

Comment:
There is probably a better way to do this but I copied from content-server for meantime.

Reply:
@vbudhram NICE FIND

Okay actually I didn't mean for that to be in all caps, apparently my caps lock was on, but I'mma leave it

Reply:
Just curious, what necessitated this change?

Reply:
Just some background, I recently made some changes [in this PR](https://github.com/mozilla/fxa/pull/17133/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519) that touches this, and was hoping a change liked this wouldn't be necessary.